src/stylesheets/core/_variables.scss

```diff
@@ -1208,7 +1208,7 @@ $box-shadow:                    0 0 2px $color-shadow !default;
 $focus-outline:                 units(0.5) solid $color-focus !default;
 $focus-spacing:                 0 !default;
 $nav-width:                     951px !default;
-$sidenav-current-border-width:  0.4rem !default; // must be in rem for math
+$sidenav-current-border-width:  units(0.5) !default; // must be in rem for math
```

Comment:
If there are user-settable vars, we should migrate them to the appropriate settings file. I see `variables` as the home for mostly non-user-settable vars.

Reply:
Should this variable be moved to another file? If so, which file?

Reply:
maybe `settings-spacing`?

Reply:
This is a component variable, not a general spacing variable. I wonder if it makes sense to live in the sidenav component file? Or to have a settings file with component vars that we pull out?

Reply:
Done in 60590cb. 

@thisisdano btw the change from `0.4rem` to `units(0.5)` is causing an issue in this code:
```
padding-left: units($theme-site-margins-mobile) - $sidenav-current-border-width;
```

It's outputting to:
```
padding-left: 1.6rem-units(.5);
```
Do you know how to get this to work?

Reply:
Fixed in 54fbd8e.